src/components/layout/Flex.js

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components'
+import {
+  flexWrap,
+  flexDirection,
+  alignItems,
+  justifyContent,
+} from 'styled-system'
+import Box from './Box'
+
+const Flex = styled(Box)(
+  {
+    display: 'flex',
+  },
+  flexWrap,
+  flexDirection,
+  alignItems,
+  justifyContent
+)
+
+// Alex, refactor the previous code to use this https://styled-system.com/api/#flexbox
```

Comment:
I guess this is for a future PR

Reply:
yes. It didn't work as the docs say, so I used what we do in UM. I don't have time to check why now, but I do want to get it the recommended way. This should not block deploying this feature since we've mentioned tagging the content for a while and showing it on different pages.